apps/etterlatte-saksbehandling-ui/client/src/utils/logger.tsx

```diff
@@ -0,0 +1,60 @@
+import Bowser from 'bowser'
+const browser: any = Bowser.getParser(window.navigator.userAgent)
+import { apiClient } from '~shared/api/apiClient'
+import { store } from '~store/Store'
+import { loggError, loggInfo } from '~store/reducers/BehandlingReducer'
+
+const defaultContext = {
+  url: window.location.href,
+  userAgent: window.navigator.userAgent,
+  userDeviceInfo: browser.parsedResult,
+  appName: 'etterlatte-saksbehandling-ui-client',
+}
+
+const loggFunc = (data: any) => apiClient.post('/logg', data)
+
+export interface IStackLineNoColumnNo {
+  readonly lineno: number
+  readonly columnno: number
+  readonly error: any
+}
+
+export const logger = {
+  info: (stackLineNoColumnNo: IStackLineNoColumnNo) => {
+    const data = { type: 'info', stackInfo: stackLineNoColumnNo, jsonContent: { ...defaultContext } }
+    loggFunc(data)
+      .then(() => store.dispatch(loggInfo(data)))
+      .catch((err) => {
+        console.error('Couldnt log info message: ', data, ' err: ', err)
```

Comment:
`Couldn\'t` ? Alternativt `Unable to ...`

Reply:
67e21b5be3716d0b971dbda3368978772a858d1c